### PR TITLE
Updates for deployment with Infrastructure Manager

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
Hi @maikenp 

Thanks for publishing this repo!

On this first PR I wanted to share the changes that I am testing to make the role work with clean deployments of Slurm with both Alma 9 and Debian 11.

Please note that I say Debian 11 as this is what's required by the [deb repo file](https://github.com/nordugrid/arc-ansible/blob/main/roles/arc_frontend/tasks/install_arc_deb.yml#L12).

This PR is currently a draft so we can continue the discussion on improvements here before merging.

Thanks,
Sebastian

### CODE REVIEW

**Comment by maikenp** [comment]
Thanks, looks all good to me. Ready when you are.

File: README.md [hunk 1]
```diff
@@ -30,26 +30,8 @@ Example:
...
-```
-
-
-
-If the group-name is different than the user-name - - make sure to put the correct one in the `group` variable. 
-
-
 ### submit_user
-What user submits the batch system job. If the user does not exist, the playbook will create the user. This user should be a normal user. It could for instance just be the default_user set above. The batch call will be issued by this user, and the job in the batch system will run as this user. 
+What user submits the batch system job. The batch call will be issued by this user, and the job in the batch system will run as this user.
```
Comment (sebastian-luna-valero) [inline README.md:36]: I think the playbook does not create the user and it must exist.

**Comment by sebastian-luna-valero** [comment]
ok, after a few more tests today, this PR is working with Alma 9 and the test x509 certificate.

If you agree, please merge and we can iterate with new PRs on other aspects.

File: README.md (hunk 1 above)
Comment (maikenp) [reply README.md:36]: Ah, yes, that might very well be true, and makes sense as this should be on the compute nodes too - I got confused with my extended playbook for setting up slurm and everything also

**Pull request merged**